src/iexamine.cpp

```diff
@@ -4395,6 +4415,21 @@ void iexamine::autodoc( player &p, const tripoint &examp )
                to_string( time_duration::from_turns( patient.activity.moves_left / 100 ) ) );
         p.add_msg_if_player( m_info, _( "The autodoc is working on %s." ), patient.disp_name() );
         return;
+    } else if( &Operator == &null_player || &Operator == &p ) {
+        uilist hmenu;
+        hmenu.text =
+            _( "Autodoc Mk. XI.  Status: Online.  WARNING: Operator missing Safety shut-down engaged... " );
+        hmenu.addentry( 1, true, 'O', _( "Override security measures." ) );
+        hmenu.query();
+
+        switch( hmenu.ret ) {
+            case 1:
+                add_msg( m_info, _( "You override the safety measures of the autodoc." ) );
+                break;
+            default:
+                return;
+                break;
```

Comment:
This is unreachable code, does the compiler complain if it's not there or something?